financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -223,17 +223,19 @@ private fun ErrorContent(
 
 @Composable
 private fun LoadedContent(
-    authenticationStatus: Async<String>,
```

Comment:
Main change here: 

- Before: Async (Can be Uninitialized, loading, success/error)
- Now: A custom status: Uninitialized,  Pending (user is in the browser / bank app), Completing, Error